utils/ansible-playbooks/deploy-monitoring/templates/ocp-monitor.j2

```diff
@@ -784,49 +786,184 @@ do
   fi
 done
 
-{% endraw %}
-{% endif %}
+# Verify master-api pods are running on all master nodes
+echo "$(datestamp) master-api Pod Status: "
+NODES=$(oc get pods -n kube-system --no-headers -o=custom-columns=POD:.metadata.name,NODE:.spec.nodeName,STATUS:.status.phase | grep -i master-api)
```

Comment:
Could we use `--selector openshift.io/component=api` instead of the grep?

Reply:
Fixed